src/util/get-github-client-config.ts

```diff
@@ -34,24 +38,28 @@ const getGitHubClientConfigFromGitHubInstallationId = async (gitHubInstallationI
 
 const getGitHubClientConfigFromAppId = async (gitHubAppId: number | undefined): Promise<GitHubClientConfig> => {
 	const gitHubServerApp = gitHubAppId && await GitHubServerApp.getForGitHubServerAppId(gitHubAppId);
-	const gitHubCloudUrls = {
+	const gitHubCloudConfig = {
 		hostname: GITHUB_CLOUD_HOSTNAME,
-		baseUrl: GITHUB_CLOUD_API_BASEURL
+		baseUrl: GITHUB_CLOUD_API_BASEURL,
+		appId: parseInt(envVars.APP_ID),
+		privateKey: PrivateKey.findPrivateKey() || ""
 	};
 
 	if (!gitHubServerApp) {
 		logger.info("No GitHub server app found. Defaulting to cloud config.");
-		return gitHubCloudUrls;
+		return gitHubCloudConfig;
 	}
 
 	const gitHubServerAppBaseUrl = gitHubServerApp.gitHubBaseUrl;
 
 	return gitHubServerAppBaseUrl
```

Comment:
Isn't this duplication of logic? you have 2 places that return `githubCloudConfig` where there only should be 1.  No need for the conditional.

```
const gitHubServerAppBaseUrl = gitHubServerApp?.gitHubBaseUrl;
return gitHubServerAppBaseUrl ? gheConfig : cloudConfig;
```